Example/Tests/Classes/ActionManagerTest.m

```diff
@@ -327,4 +308,20 @@ - (void)test_active_period_true
     XCTAssertTrue(result.matchedActivePeriod);
 
 }
+
+#pragma mark Helpers
+
+-(NSDictionary *)_configForActivePeriodMatch:(BOOL)activePeriod
```

Comment:
objc doesn't use _ syntax. Private methods are determined by not exposing it in .h file. 
Also the method name needs to make the parameter very clear. This is one part we can improve across the SDK in general. Right now the parameter seems like it will take some sort of Match. I suggest something like:
`messageConfigInActivePeriod:(BOOL)inActivePeriod`